src/main/antora/modules/websvcs/pages/xml-websvcs/jakarta-xml-binding.adoc

```diff
@@ -0,0 +1,6 @@
+== Jakarta XML Binding
+
+Jakarta XML Binding (JAXB) provides a way to convert Java objects to XML and vice versa. It is a part of the Jakarta EE platform.
```

Comment:
The first sentence is good, but the 2nd one is redundant (anything in here is a part of the platform). Can you add something like this?

> JAXB can be used independently or in combination with Jakarta REST or Jakarta XML Web Services, in which case it provides a standard data binding for web service messages.

Also, can you link to Jakarta REST  and  Jakarta XML Web Services within the current tutorial?